src/hooks/useQuestionnaireOptions.ts

```
@@ -0,0 +1,42 @@
+        },
+      }),
+    })),
+  });
+
+  const allQuestionnaireOptions = queries.reduce((acc, queryResult) => {
+    if (queryResult.data?.results) {
+      const newOptions = queryResult.data.results.map((q) => ({
+        slug: q.slug,
+        title: q.title,
+      }));
+      return [...acc, ...newOptions];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)